ush/templates/FV3LAM_wflow.xml

```diff
@@ -261,7 +261,7 @@ define resources used for each tasks
 <!ENTITY WALLTIME_ANAL_GSI  "{{ wtime_run_anal }}">
 
 <!ENTITY WALLTIME_FCST_SPINUP  "{{ wtime_run_fcst_spinup }}">
-{%- if machine in ["JET", "HERA"] %}
+{%- if machine in ["JET"] %}
```

Comment:
Why "HERA" is removed?

Reply:
HERA has only one kind of node. It is more like Orion and WCOSS2.
Hera does not need to set like JET, which has many kind of nodes.

Reply:
In the latter case, do we need to manually compute layout_x*layout_y + write_group * tasks_per_group and then decide an appropriate `nnodes_run_fcst ` value?

Reply:
Yes, we do need to manually calculate model layout and nodes needed for a run.